region/maridia/inner-yellow/Pseudo Plasma Spark Room.json

```diff
@@ -300,6 +300,22 @@
         ]}
       ]
     },
+    {
+      "link": [2, 2],
+      "name": "Skulltera Ice Moonfall Door Lock Skip",
```

Comment:
So I did this one and fell into the center box of Crab Shaft and was softlocked.

Reply:
Yeah I found that for a lot of these moonfalls, holding an angle button (or up) through the transition is helpful to expand your hitbox, too avoid falling too deep in the next room. In the examples I tested, that seemed to be enough to prevent getting stuck.

In principle the way to model this accurately would be to put an exitCondition on these strats, with a leavesWithStoredFallSpeed or something similar. Then strats on the bottom room would have a corresponding entrance condition, to take into account the effects of the fall speed (either positively or negatively). But then we'd need a way to put specific speed values in those exit/entrance conditions. Also the precise position through the transition (horizontal and vertical) could matter, and that could get messy fast. And I'm not sure if we have any examples yet where it would actually matter logically.